### PR TITLE
feat: update company logos collection cards

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksupdate-external-product-logos-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/inbox/1199634971443477/1204108582255298/1204168041710465) 🎟️

## 🗒️ What

This PR updates the company logos for the collection cards to be 'themed.' 

Screenshot 2023-03-20 at 3 01 23 PM

## 🛠️ How

I took a similar approach as in https://github.com/hashicorp/dev-portal/pull/1752

## 🧪 Testing

- [Visit the preview](https://dev-portal-git-ksupdate-external-product-logos-hashicorp.vercel.app/terraform/tutorials)
- View the cards, toggle between light and dark to validate that they still work.

### CODE REVIEW

File: src/views/product-landing/components/overview-cta/overview-cta.module.css [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 .textPart {
 	flex: 1 1 0;
-	min-width: min(100%, 300px);
+	min-width: min(100%, 260px);
```
Comment (kendallstrautman) [inline src/views/product-landing/components/overview-cta/overview-cta.module.css:17]: Oops these should be on the other PR

File: src/components/collection-card/components/company-logo/index.tsx [hunk 2]
```diff
@@ -14,26 +15,58 @@ export default function CompanyLogo({ name }: CompanyLogoProps): JSX.Element {
...
+			<ThemedLogo
+				hideOnTheme={GlobalThemeOption.dark}
+				src={src.light}
+				name={name}
+			/>
+		</span>
+	)
+}
+
+function ThemedLogo({
```
Comment (kendallstrautman) [inline src/components/collection-card/components/company-logo/index.tsx:45]: Could add a note here as a breadcrumb. 

Eventually we will lift up a global `ThemedImage` component. Took the skateboard approach here and duplicated the approach from https://github.com/hashicorp/dev-portal/pull/1752 for this specific context.

**Review by kellyschummer — APPROVED** [review]
lgtm!! 🏅

**Review by ashleemboyer — CHANGES_REQUESTED** [review]
This PR includes more than logo changes in collection cards -- also includes changes to every product landing page. Can the two be separated for ease of reviewing? 🙏🏻

**Review by ashleemboyer — COMMENTED** [review]
Functionality is looking good, have a couple of questions for clarity! 🙋🏻

- Is there a naming convention we're using for the image files? I see a mix of using hyphens and underscores, so just want to check.
- What does `_dm` signify? Dark mode?

**Comment by ashleemboyer** [comment]
> @ashleemboyer yes _dm signifies dark mode. Would you prefer that I update to use a hyphen for this suffix?

@kendallstrautman I see value in using one or the other for consistency, so since hyphens are used on the existing file names I think hyphens would be good! What are your thoughts on spelling out the `dm` abbreviation for clarity?

**Comment by kendallstrautman** [comment]
Sure I can do that! 👍🏻

**Comment by kendallstrautman** [comment]
@ashleemboyer I've updated the asset names! lmk if that looks good to you

**Review by zchsh — APPROVED** [review]
✅ I checked out the preview, looks great!

**Comment by kendallstrautman** [comment]
@ashleemboyer let me know if you still have blocking feedback here 👐🏻

**Comment by ashleemboyer** [comment]
> @ashleemboyer let me know if you still have blocking feedback here 👐🏻

@kendallstrautman just now getting a moment to re-review! Taking a look now 😊

**Pull request merged**